src/wallet/db.cpp

```diff
@@ -167,8 +167,8 @@ bool CDB::Recover(const std::string& filename, void *callbackDataIn, bool (*reco
     // Call Salvage with fAggressive=true to
     // get as much data as possible.
     // Rewrite salvaged data to fresh wallet file
-    // Set -rescan so any missing transactions will be
-    // found.
+    // call rescanblockchain (RPC) so any missing
```

Comment:
It happens outside of this call (somewhere in wallet.cpp). But the comment doesn't say "It" calls `rescanblockchain`, it either say "you" can call `rescanblockchain()`.